xm-commons-data-spec/src/test/java/com/icthh/xm/commons/domain/TestInputDataSpec.java

```diff
@@ -0,0 +1,43 @@
+package com.icthh.xm.commons.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import lombok.Data;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"key", "path", "inputSpec", "inputForm", "wrapResult", "anonymous", "txType",
+    "tags", "httpMethods"})
+@Data
+public class TestInputDataSpec implements SpecWithInputDataAndForm {
+
+    @JsonProperty("key")
+    private String key;
+
+    @JsonProperty("inputSpec")
+    private String inputSpec;
+
+    @JsonProperty("inputForm")
+    private String inputForm;
+
+    // TODO test, that this field are ignored in json
+    @Override
+    public String getInputDataSpec() {
```

Comment:
how about 
`
String inputDataSpec();
void inputDataSpec(String inputSpec);
`

Reply:
IMO, it looks like method overloading more
but this methods implements different logic getting and setting objects

and this's a test class, changes will come up here soon :)

Reply:
specification schema changed according to 
![Uploading image.png…]()